user_guide_src/source/models/model.rst

```diff
@@ -141,6 +141,8 @@ This array should be updated with the field names that can be set during ``save(
 against just taking input from a form and throwing it all at the model, resulting in
 potential mass assignment vulnerabilities.
 
+.. note:: The ``$primaryKey`` should never be an allowed field.
```

Comment:
I think it should be primary key field as term, not variable with $ prefix

Reply:
I mean `$primaryKey` value.
`$primaryKey` is probably primary key field, but it can be other unique key.

Reply:
how about change to:

```suggestion
.. note:: The ``primary key field`` should never be an allowed field.
```

Reply:
The text decoration ` `` ` is for source code (variables, method names, classnames).

How about this?

```suggestion
.. note:: The ``$primaryKey`` field should never be an allowed field.
```

Reply:
Yes, that seems better 👍